pkg/rulemanager/cel/cel.go

```
@@ -64,6 +64,13 @@ func NewCEL(objectCache objectcache.ObjectCache, cfg config.Config, mm ...metric
 		k8s.K8s(objectCache.K8sObjectCache(), cfg),
 		containerprofile.CP(objectCache, cfg, mm...),
 		containerprofilenetwork.CPNetwork(objectCache, cfg, mm...),
+		// Deprecated backward-compat aliases for the ap.*/nn.* CEL namespaces
+		// renamed to cp.* in #864 (no aliases were kept at the time, which
+		// silently disabled any pre-existing user rule still using the old
+		// names). Retained for a transition window; remove once user rules
+		// have migrated to cp.*.
+		containerprofile.AP(objectCache, cfg, mm...),
```

golangci-lint (2.12.2)

[error] SA1019: containerprofile.AP is deprecated: kubescape/node-agent#864 renamed the ap.*/nn.* CEL helper namespaces to cp.* with no backward-compatible aliases, which silently disabled any pre-existing user-authored CEL rule still referencing the old names (pkg/utils/cel.go logs a warning and just skips the rule on compile failure). AP restores those names for a transition window: it shares the same containerProfileLibrary instance and the same funcSpecs table as cp.*, so every ap.* call is wired to the exact same Go implementation (l.wasExecuted, l.wasPathOpened, ...) as its cp.* equivalent — only the CEL-facing function name and overload id differ (cel-go requires overload ids to be unique per environment, so ap.* can't literally reuse cp.*'s FunctionOpt values). Remove once user rules have migrated to cp.*. 

(staticcheck)